content/en/docs/ops/ambient/usage/helm-installation/index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Installing Ambient Mesh with Helm
```

Comment:
```suggestion
title: Install Ambient Mesh with Helm
```

Reply:
Can we have a separate Install section under Operations>Ambient Mesh, and then move this under there. Also it would make sense to remove the other empty Installation Guide page: https://deploy-preview-13871--preliminary-istio.netlify.app/latest/docs/ops/ambient/usage/install/

Reply:
Makes sense, I'll fix it. Thanks!

Reply:
@kfaseela Could you check this new section format? https://deploy-preview-13871--preliminary-istio.netlify.app/latest/docs/ops/ambient/install/